ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml

```
@@ -28,6 +32,18 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
   to: operator-registry
+- dockerfile_path: operator-framework-cli.Dockerfile
+  inputs:
+    ocp_4.16_base-rhel9:
+      as:
+      - registry.ci.openshift.org/ocp/4.16:base-rhel9
+    ocp_builder_rhel-8-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.21-openshift-4.16
+    ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
+  to: operator-framework-cli
```

What do you think about pluralizing this name?
```suggestion
  to: operator-framework-clis
```